### PR TITLE
[onert] OperationValidator for ArgMax

### PR DESCRIPTION
**Pull request by hseok-oh**
- Enable OperationValidator for ArgMax operation
- Add negative test for ArgMax
- Enable ArgMax test for int64 output

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: runtime/onert/core/src/ir/OperationValidator.cc [hunk 1]
```diff
@@ -84,6 +84,18 @@ void OperationValidator::visit(const operation::AddN &node)
   }
 }
 
+void OperationValidator::visit(const operation::ArgMax &node)
+{
+  const auto input_index(node.getInputs().at(operation::ArgMax::Input::INPUT));
```
Comment (ragmani) [inline runtime/onert/core/src/ir/OperationValidator.cc:89]: How about adding checking axis type too?

Comment (hseok-oh) [reply runtime/onert/core/src/ir/OperationValidator.cc:89]: I've updated to check axis type. PTAL

**Comment by hseok-oh** [comment]
@ragmani @glistening PTAL

**Review by ragmani — APPROVED** [review]
LGTM

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**